modules/ROOT/pages/flex-gateway-register.adoc

```diff
@@ -0,0 +1,26 @@
+= Register a Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+Registration is used for licensing, metering and billing. You must register every flex gateway regardless of how you want to manage it.
```

Comment:
Registration is used for licensing, metering and billing > Flex Gateway uses the registration for licensing, metering, and billing purposes. 

You must register > Register ...

Reply:
Changed o "Registration allows for licensing, metering and billing..."